content/blog/2021-03-08-week-1.md

```diff
@@ -3,27 +3,27 @@ title: 'Week 1 in review'
 date: 2021-03-08
 ---
 
```

Comment:
Add an introductory sentence or two to let the reader know what's going on. Something like:
```suggestion
After months of preparation, this week the Feminizmap project kicked off! The first week ... 
```